### PR TITLE
Support credentials

### PR DESCRIPTION
**Pull request by swethalohith2**
## What

Add credential Support

- Add get credential
- Add create credential
- Add delete credential

## References

[Kanban Card](https://instana.kanbanize.com/ctrl_board/132/cards/129239/details/)

### CODE REVIEW

File: synctl
Comment (rongzhus) [inline synctl:1025]: only one column to show for creds, I think we can remove the title `CREDENTIALS`. It is easy to confuse it with credentials. For example:
```
CREDENTIALS
BIN_PASS4
BIN_PASS3
BIN_PASS2
USER_NAME_TEST
BIN_PASS1
BIN_PASS5
```

Comment (rongzhus) [reply synctl:1025]: The list is unsorted, it is not easy to find if the list is very long. can you sort it alphabetically？

**Comment by rongzhus** [comment]
Remember to update README and usage for new command.

**Comment by rui1234** [comment]
Do not merge the PR until R252 is released since creds are only supported since R252

File: README.md
Comment (rui1234) [inline README.md:309]: Let us change it to Manage Credentials since not only querying

File: synctl
Comment (rongzhus) [inline synctl:2384]: remove `--key` here, we use same syntax with `synctl delete lo <location-id1> ...`, for creds, the command should be `synctl delete cred <cred-name> <cred-name2> ...`, get_args.id can get multiple key name. When get_args.delete_type is SYN_CRED, get_args.id should not be none, and at least one key-name. you can refer this pop_instance.delete_synthetic_locations(get_args.id).

File: synctl
Comment (rongzhus) [inline synctl:1443]: can you put the delete request under the if clause, only if cred in credentials list, synctl will send a delete request, otherwise exited.

**Review by rongzhus — APPROVED** [review]
Looks good to me.

**Comment by rongzhus** [comment]
can we merge this PR after conflicts are resolved? @rui1234

**Review by rui1234 — APPROVED** [review]

**Pull request merged**
